src/components/settings/LocationSettings.js

```diff
@@ -149,5 +151,9 @@ export const LocationSettings = () => {
 const styles = StyleSheet.create({
   containerStyle: {
     marginBottom: normalize(21)
+  },
+  mapContainer: {
+    height: normalize(250),
+    width: '100%'
```

Comment:
is this width really needed? i tried without it and nothing changed.
the problem of rendering the map in the small width needs to be tackled. the width here seems to not have any impact.

Reply:
you are right, it's not needed and will be removed in the upcoming commit.